### ISSUE TITLE
Decluttering the context menu on both platforms

### ISSUE THREAD

**Issue by vrindavan** [brave/brave-browser#16138]
One example could be, if we are not using and turned off brave rewards then let it be in settings only. Remove from context menu.

**Comment by godly-devotion** [comment]
I agree. The settings menu could use some cleanup, mainly in the order of the items displayed and just having too much stuff available that aren't used often.

Things that could be removed:
- **Sync**: Can be honestly inside Settings. How often would the user want to go into Sync from the menu to adjust something?

- **Brave Ad Block**: When I first started using Brave, I couldn't find where the built-in Ad blocker rules where when I went into Settings, which is where I expected this option to be in. Also, this is more of a set once and forget section and probably does not get visited often.

- **Create a New Profile**: Not something done often. Can also live in Settings.

- **Edit**: I have never used this option, mainly because it requires extra steps to get here. Keyboard shortcuts and right click context menu already provides Cut, Copy, and Paste with less steps. On a Mac, you can even go to the Edit menu to see these common options.

Personal opinion on things that could be further simplified:
- **Brave Rewards**: Honestly there is a whole Settings page dedicated to the Rewards section and I personally have not turned this on. If the user hasn't turn this on, I wouldn't want to see it here either.

- **Wallet**: I don't do crypto (probably like a lot of users) and don't need a prominent section for that in a web browser.

- **Open Guest Window**: I just tell guests to use Private Window which does almost the exact same thing.

Individually it might not be a lot of menu items, but altogether I think it will help cleanup the menu that hasn't been touched since the early days of Chrome.

**Comment by vrindavan** [comment]
@godly-devotion thanks a lot for mentioning all of that. I didn't have time for that.

Yes, exactly. It would look much better.
